curl/src/client.cc

```diff
@@ -312,6 +317,19 @@ namespace leatherman { namespace curl {
         }
     }
 
+    void client::set_client_protocols(context& ctx) {
+        if (!_client_protocols) {
+            // If not set, allow all protocols
+            _client_protocols = CURLPROTO_ALL;
+            LOG_WARNING("No client protocol was set. Allowing all protocols.", _client_protocols);
```

Comment:
The `_client_protocols` argument is unused in this log message. I'd drop the argument.